docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+Splitting the global mark phase into these increments and subincrements reduces pause times by running the majority of the GC work concurrently and interleaving global mark phase increments with partial GC cycles, and, rarely, [global GC cycles](vgclog_examples.md#balanced-global-gc-cycle).
+
+A global *mark* GC cycle increment is triggered by an [allocation taxation threshold](vgclog_examples.md#balanced-examples).
+
+The following elements log the GC increments, subincrements and operations of the global *mark* GC cycle:
+
+|GC increment         | GC operations| *stop-the-world* or concurrent| XML element of GC increment| Details                         |
+|---------------------|-------------|-------------------------------|--------------------------------------|-----------------------|
+|`global mark phase` subincrement| mark | *stop-the-world* | `<gc-start>`, `<gc-end>` |The global mark phase operations start at the beginning of the cycle and run through all *regions* until the final *region* |
+|`GMP work packet processing` subincrement| work packet processing (WPP) operations | concurrent and sometimes final operations during a *STW* to complete the subincrement | `<concurrent-start>`, `<concurrent-end>`| The `GMP work packet processing subincrement` runs immediately after the `global mark phase` subincrement |
+|final global mark phase increment | final global mark phase operations including class unload | *stop-the-world* | `gc-start>`, `<gc-end>`| Final increment. Runs the final global mark phase operations followed by operations to finish the cycle  |
+
+The `balanced` global *mark* GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and some child elements are omitted for clarity:
+
+```xml
+<exclusive-start/>                        (STW pause starts)
+
+<allocation-taxation/>                    (memory threshold trigger recorded)
+
+<cycle-start type="global mark phase"/>   (global mark cycle starts)
+
+<gc-start type="global mark phase"/>      (1st GMP STW subincrement starts)
+
+    <mem-info>                            (memory status before operations)
+
+    <remembered-set>
+
+    </mem-info>         
+
+</gc-start> 
+
+<gc-op type="mark increment" />           (STW copy forward operation completed)
+
+<gc-op  type="class unload" />            (optional STW class unload operation completed)
+
+<gc-end>                                  (partial cycle increment ends)
```

Comment:
'partial' cycle is confusing here (this is not PGC cycle). this is end of 1st GMP STW subincrement

Reply:
Good spot, this is a copy error.